kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/create-domain-script.sh

```diff
@@ -10,4 +10,6 @@ export DOMAIN_HOME=${DOMAIN_HOME_DIR}
 # Create the domain
 wlst.sh -skipWLSModuleScanning ${CREATE_DOMAIN_SCRIPT_DIR}/create-domain.py
 
-
+if [ "${ISTIO_ENABLED}" == "true" ]; then
+  wlst.sh -skipWLSModuleScanning ${CREATE_DOMAIN_SCRIPT_DIR}/istio-fix-domain.py
```

Comment:
seems like there should be some error checking in here?

Reply:
The wlst exception will be thrown to the pod logs if there is any.